ci/integration_tests/src/tests/cw20_stake_test.rs

```diff
@@ -0,0 +1,130 @@
+use core::time;
+use std::thread;
+
+use crate::helpers::{chain::Chain, helper::create_dao};
+use cosm_orc::config::SigningKey;
+use cosmwasm_std::{to_binary, Uint128};
+use cw20_stake::{msg::StakedValueResponse, state::Config};
+use cw_core_interface::voting::VotingPowerAtHeightResponse;
+use test_context::test_context;
+
+// #### ExecuteMsg #####
+
+#[test_context(Chain)]
+#[test]
+fn execute_stake_tokens(chain: &mut Chain) {
+    let key: SigningKey = chain.key.clone().try_into().unwrap();
+    let account = key
+        .public_key()
+        .account_id(&chain.cfg.chain_cfg.prefix)
+        .unwrap();
+    let voting_contract = "cw20_staked_balance_voting";
+
+    let res = create_dao(chain, None, "exc_stake_create_dao", account.to_string());
+    assert!(res.is_ok());
+
+    let dao = res.unwrap();
+
+    let voting_addr = dao.state.voting_module.as_str();
+
+    // stake dao tokens:
+    chain
+        .orc
+        .contract_map
+        .add_address(voting_contract, voting_addr)
+        .unwrap();
+    let staking_addr = &chain
+        .orc
+        .query(
+            voting_contract,
+            "exc_stake_q_stake",
+            &cw20_staked_balance_voting::msg::QueryMsg::StakingContract {},
+        )
+        .unwrap()
+        .data
+        .unwrap();
+    let staking_addr: String = serde_json::from_slice(staking_addr.value()).unwrap();
+
+    chain
+        .orc
+        .contract_map
+        .add_address("cw20_stake", staking_addr.to_string())
+        .unwrap();
+    let res = chain
+        .orc
+        .query(
+            "cw20_stake",
+            "exc_stake_q_cfg",
```

Comment:
this string name doesnt match up to the query?

Reply:
Good catch!